app/cafe.py

```diff
@@ -0,0 +1,18 @@
+import datetime
+from app.errors import (NotVaccinatedError,
+                        OutdatedVaccineError,
+                        NotWearingMaskError)

```

Comment:
According to the checklist, you should use `( )` while importing multiple classes/modules. Please, consider changing the import style.